scripts/development/start-dev.sh

```diff
@@ -1,13 +1,105 @@
- #!/bin/bash
+#!/bin/bash
+# export COMPOSE_HTTP_TIMEOUT=300
+# sh scripts/development/start-dev.sh
 
-if [ ! -f .env ]; then
-    cp .env_example .env
-fi
 
-echo "Building, Starting and Provisioning TruBudget for Development"
+Help()
+{
+    # Display Help
+    echo
+    echo "Help"
+    echo "This script setups a TruBudget Development instance of your choice."
+    echo "Frontend, API, Excel-Export, Email-Notification, Storage-Service are always hot reloaded on save."
+    echo "The Multichain data is NOT persisted and will be lost after a restart."
+    echo "The environmental variables are set automatically."
+    echo
+    echo "Hint: Make sure you are in the root directory of TruBudget."
+    echo
+    echo "Syntax: sh scripts/development/start-dev.sh [options]"
+    echo "Example: sh scripts/development/start-dev.sh --full --build"
+    echo
+    echo "options:"
+    echo "      --slim                  Starts a TruBudget instance with master-node, master-api, provisioning and frontend."
+    echo "      --full                  Starts a TruBudget instance with master-node, emaildb, minio, master-api, email-service,"
+    echo "                              provisioning, excel-export-service, storage and frontend."
```

Comment:
extend the info by the default behaviour. I don't know what the script is doing without an option- would expect Trubudget without any service enabled

Reply:
Without any option, I would expect a full instance

Reply:
done - added:"Default option: full TruBudget instance with all services in debug mode"